cuegui/cuegui/Constants.py

```diff
@@ -32,7 +32,13 @@
 import opencue
 
 
-VERSION = "1.3.0"
+possible_version_paths = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), 'VERSION.in')
```

Comment:
Tests are failing on this PR -- check out the status checks below -- and it looks like it's because this variable is named `possible_version_paths` instead of `possible_version_path`.

The PR status checks have links to our Azure DevOps project where you can see logs from the tests. You can also run tests locally by running `python setup.py test` from the `cuegui/` directory.

Reply:
Done.